src/app.py

```diff
@@ -16,13 +25,21 @@
     "entity_name_long": "license plate",
     "entity_name_short": "License #",
     "data_source": "https://data.seattle.gov/City-Business/Active-Fleet-Complement/enxu-fgzb",
-    "lookup_url": "license-lookup",
+    "lookup_url": "license",
+    "query_param": "license"
 }
 
 
-@app.route("/")
+@app.route("/license")
 def license_page():
-    """Displays the homepage."""
+    """License lookup form render"""
+    # Check for badge query params and load if it exists.
+    license = request.args.get('license')
+    if license:
+        html = dataset.license_lookup(license)
+        return render_template("index.html", **LICENSE_CONTEXT, entity_html=html)
+
+    # Return the basic lookup form if not
     return render_template("index.html", **LICENSE_CONTEXT)
```

Comment:
For each of these cases, I'd suggest the following change for simplicity:

```suggestion
    license = request.args.get('license')
    html = ""
    if license:
        html = dataset.license_lookup(license)

    return render_template("index.html", **LICENSE_CONTEXT, entity_html=html)
```

This only hydrates the extra HTML if it's available, and still renders everything well when it isn't.

Reply:
Oh this makes sense, I didn't think about that. I'll make those changes.